examples/rugby_ball/driver.f90

```diff
@@ -122,6 +122,9 @@ program usrneko
        ! they're within a reasonable range unfortunately this takes a bit of
        ! trial and error, but we should include a subroutine that
        ! allows us to rescale
+       ! call cmult(df0dx,0.01_rp,n)
+       call cmult(dfdx,100.0_rp,n)
+       fval(1) = fval(1)*100.0_rp
        ! (and also prints out some norms to make the trial and error and
        ! bit easier)
```

Comment:
dont we have some bounds on what the norm is allowed to be? I remember casper
and niels talking about that.

Reply:
This one is based on the recommendation by Svanberg:
"the user should then preferably scale the constraints in such a way that 1 ≤ gmax i ≤ 100 for each i" and that here we're using a volume constraint, so it will be $\in [0,1]$.

Regarding the objective function, I have seen people in the past check their first iteration, scale everything to 10, and remember that scaling for the rest of the optimization problem.
I believe that's what's done in Niels' petsc code.